### ISSUE TITLE
cool stuff

### ISSUE THREAD

**Issue by Talv** [chansey97/sc2-mined-data#1]
Hey

That's a really interesting stuff you got there.
I wonder how did you pull that from the executable? I guess this has something to do with this build being *quite* old, but I'm still curious how hard was it? Was the data well structured?

Because I've been looking at current builds of SC2 through IDA, and the level of complexity in regards to data structures is quite insane.

**Comment by chansey97** [comment]
Yes. The decompiled code of early builds is well structured (no packed), so it is easy to pull data. The decompiled code structure of current builds has been changed (possibly packed), so it is difficult to crack.
